packages/tokens/src/tokens/font/typography.json

```diff
@@ -80,7 +80,7 @@
       "fontFamily": "{vads-font-family-serif-regular}",
       "fontSize": "{vads-font-size-heading-level-1}",
       "lineHeight": "{vads-font-line-height-heading-2xl}",
-      "marginBottom": "{vads-space-3xl}",
+      "marginBottom": "{vads-space-lg}",
```

Comment:
Just for personal edification, did this change in Figma? I feel like as part of the initial outlay of the typography tokens (with the base font tokens) these were much larger for the Headings.

Reply:
I think they may have but maybe @jessicawoodin can confirm